src/axom/inlet/LuaReader.cpp

```diff
@@ -11,28 +11,24 @@
  *******************************************************************************
  */
 
+# include <fstream>
+
 #include "axom/inlet/LuaReader.hpp"
 
 #include "axom/core/utilities/FileUtilities.hpp"
 #include "axom/core/utilities/StringUtilities.hpp"
+#include "axom/inlet/inlet_utils.hpp"
 
 #include "fmt/fmt.hpp"
 #include "axom/slic.hpp"
 
+#define SOL_ALL_SAFETIES_ON 1
```

Comment:
We do have specific changes to the sol.hpp.  its hard to see them in this PR because the file is too long.  but if you view the sol.hpp file and search for axom you can see them there.

Reply:
Can you move this line into sol.hpp with the other defines we had to do?